src/plane.js

```diff
@@ -0,0 +1,19 @@
+class Plane {
+    id;
+
+    constructor(id) {
+        this.id = id;
+    }
+
+
+}
+
+module.exports = Plane;
+
+
+// Test to check that planes cannot take off when they are not at the airport
+// const planeStatus = 'notAtAirport';
+
+// if (planeStatus === 'notAtAirport') {
+//   assert.strictEqual(takeOff(), false, 'Cannot take off when plane is not at the airport')
+// }
```

Comment:
Had googled this but not used. Was thinking of putting plane status into the Plane Class. Happy with my approach in the airport.